sim-cli/src/main.rs

```diff
@@ -15,34 +14,51 @@ use simple_logger::SimpleLogger;
 #[derive(Parser)]
 #[command(version)]
 struct Cli {
-    #[clap(index = 1)]
-    sim_file: PathBuf,
+    /// Path to folder containing simulation files and where simulation results will be written
+    #[arg(index = 1, env = "SIMLN_DATA_DIR")]
+    data_dir: PathBuf,
+    /// Name of the simulation file to use. These are located in `${SIMLN_DATA_DIR}/sims/`
+    /// and are in JSON format. If not specified, the cli will prompt the user to select one.
```

Comment:
Small personal preference to have:
*  `sim-file` in `SIM_LN_DATA_DIR` 
* simulation results in `SIM_LN_DATA_DIR/results` sub-dir

Reasoning being:
* `simulation_xyz.csv` files can get a bit noisy when you've run the simulator a few times. 
* Sim file in a sub-dir seems a little atypical (I can see folks creating a sim file in the root, trying to run then needing to move it).